docs/config.md

```diff
@@ -93,6 +94,10 @@ This can be achieved in a few ways:
   - This field takes precedence over `storage.docdb.mongo-server-url` and `MONGO_SERVER_URL` env var.
   - The value should point to a directory that has a file named `MONGO_SERVER_URL` that contains the env var. Each time the file is updated, the new value will be read.
   - One common use case is to store the value of `MONGO_SERVER_URL` in a secret with the key `MONGO_SERVER_URL` and mount the secret at the path specified in this field. When the secret is updated, the new value will be fetched by Tekton Chains.
+- Setting the value of `storage.docdb.mongo-server-url-path` field
+  - This field takes precedence over `storage.docdb.mongo-server-url-dir`, `storage.docdb.mongo-server-url` and `MONGO_SERVER_URL` env var.
+  - This field should point to the file path that contains the mongo server url. Each time the file is updated, the new value will be read
+  - One common use case is to store the value of mongo server url in a secret with any key and mount the secret at the path specified. When the secret is updated, the new value will be fetched by Tekton Chains
 
```

Comment:
It would be helpful if you create a table which shows precedence of using `storage.docdb.mongo-server-url-dir`, `storage.docdb.mongo-server-url` and `MONGO_SERVER_URL`

Reply:
And also we should make a recommendation to users on which approach is recommended. For example, `storage.docdb.mongo-server-url` should be avoided since credentials are stored in a ConfigMap instead of a Secret.

Reply:
updated!